### PR TITLE
Add authentication support, refactor

### PR DESCRIPTION
**Pull request by grongor**

### CODE REVIEW

File: tests/KafkaTestCase.php [hunk 1]
```diff
@@ -11,11 +11,12 @@
 use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
 use Symfony\Component\Yaml\Yaml;
 use function Safe\file_get_contents;
+use function Safe\sprintf as sprintf;
 use function sys_get_temp_dir;
 
 abstract class KafkaTestCase extends TestCase
 {
-    protected function createYamlBundleTestContainer() : Container
+    protected function createYamlBundleTestContainer(string $config) : Container
```
Comment (simPod) [inline tests/KafkaTestCase.php]:
```suggestion
    protected function createYamlBundleTestContainer(string $configFileName) : Container
```

File: src/Kafka/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\KafkaBundle\Kafka;
+
+use function Safe\gethostname;
+use function Safe\sprintf;
+
+class Configuration
```
Comment (simPod) [inline src/Kafka/Configuration.php]: final

File: tests/KafkaTestCase.php (hunk 1 above)
Comment (grongor) [reply tests/KafkaTestCase.php]: není to celý filename

Comment (simPod) [reply tests/KafkaTestCase.php]: configname

**Pull request merged**